Doc/library/compression.rst

```diff
@@ -0,0 +1,18 @@
+The :mod:`!compression` package
+===============================
+
+.. versionadded:: 3.14
+
+.. attention::
+
+   The :mod:`!compression` package is the new location for the data compression
```

Comment:
"New" always seems such a temporal term.  I'm just spitballing here, but how about something like: "The compression package provides a unified interface to several data compression routines. Some of these have historically been available as separate modules; those will continue to be available under their original names for compatibility reasons, and will not be removed without a deprecation cycle".

Reply:
Hm, I definitely understand "new" may not be the right word here. I worry if we say "a unified interface to ... compression routines" that people will expect `compression` to hold some high level `compress` function that abstracts over the various implementations, which is not the case.

Perhaps
"The compression package contains the canonical compression modules containing interfaces to several different algorithms. Some of these have historically been available as separate modules; those will continue to be available under their original names for compatibility reasons, and will not be removed without a deprecation cycle. The compression.* import names are encouraged for use where practical."

Reply:
I pushed up a slightly tweaked version of my suggestion. Please let me know what you think!

Reply:
the intro looks good